apps/coscrad-frontend-e2e/src/e2e/resource-index-to-detail-flows/tag-term.e2e.cy.ts

```diff
@@ -0,0 +1,54 @@
+describe(`tagging a note`, () => {
+    const termBaseRoute = `/Resources/Terms/`;
+
+    const termId = `9b1deb4d-3b7d-4bad-9bdd-2b0d7b110003`;
+
+    const tagId = '9b1deb4d-3b7d-4bad-9bdd-2b0d7b110004';
+
+    const tagResourceLabel = `Tag Resource or Note`;
+
+    describe(`when tagging a term`, () => {
+        beforeEach(() => {
+            cy.visit(`${termBaseRoute}${termId}`);
+
+            cy.login();
+
+            cy.getByDataAttribute('nav-menu-icon').click();
+
+            cy.get('[href="/Resources"] > .MuiButtonBase-root').click();
+
+            cy.getByDataAttribute('term').click();
+
+            cy.get(`[data-testid="${termId}"] > :nth-child(1)`).click();
```

Comment:
We might want to create the term from scratch and then just click on the first (and only since we'll also clear the db in-between) term.

Reply:
We need to do that as soon as the infrastructural support is there.